src/templates/tr/error.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base/error.html" %}
+
+{% block unknown_error %}Bilinmeyen Hata{% endblock %}
+
+{% 
+  set localizedStatusCode = {
+    400: "Geçersiz istek",
+    404: "Sayfa Bulunamadı",
+    500: "İç Sunucu Hatası",
+    502: "Geçici Sunucu Hatası"
+  }
+%}
+{% macro expandedTitle(errorTitle) %}Hata: {{ errorTitle }}{% endmacro %}
+{%
+  set localizedErrorMessage = {
+    "Unsupported year requested": "Desteklenmeyen yıl talep edildi",
+    "Unsupported chapter requested": "Desteklenmeyen bölüm talep edildi",
+    "Unsupported language requested": "Desteklenmeyen dil talep edildi",
+    "Not Found": "İstenen URL, server üzerinde bulunamdı. Eğer URL'i manuel olarak girdiyseniz, lütfen yazımınızı kontrol edin ve tekrar deneyin."
+  }
+%}
+{% macro expandedErrorMessage(errorCode, errorDescription, errorMessage) %}
+{{ errorCode }} {{ errorDescription }}: {{ errorMessage }}
```

Comment:
This will actually be the English error description. The Turkish will be above, but we include the original as we don't translate all the descriptions as he hope never to see most of them!

```suggestion
{{ errorCode }} <i lang="en">{{ errorDescription }}</i>: {{ errorMessage }}
```